app/swagger/requests/prescriptions.rb

```diff
@@ -0,0 +1,306 @@
+# frozen_string_literal: true
+module Swagger
+  module Requests
+    class Prescriptions
+      include Swagger::Blocks
+
+      swagger_path '/v0/prescriptions' do
+        operation :get do
+          key :description, 'Get a list of active and inactive prescriptions'
+          key :operationId, 'indexPrescriptions'
+          key :tags, %w(prescriptions)
+
+          parameter name: :page, in: :query, required: false, type: :integer,
+                    description: 'Page of results, greater than 0'
+
+          parameter name: :per_page, in: :query, required: false, type: :integer,
+                    description: 'number of results, between 1 and 99'
+
+          parameter name: :sort, in: :query, required: false, type: :string,
+                    description: "Comma separated sort field(s), prepend field(s) with '-' for descending sort"
+
+          parameter name: :filter, in: :query, required: false, type: :string,
```

Comment:
Some of these might end up being common components across a variety of models and could later be included in a more modular way?